NEW_README.md

```diff
@@ -19,15 +19,34 @@ RUN is a task runner that simplifies executing and managing tasks defined in `ta
 
 RUN can be installed as a single binary or via the Go command line tool:
 
-### **Pre-compiled binary:**
+### Pre-compiled binary
 
 Download the latest release from the [releases page](https://github.com/amonks/run/releases), and extract the binary to a directory in your `PATH`. This can be done in a single command like:
 
     $ curl -sL https://github.com/amonks/run/releases/download/<RELEASE_VERSION>/run_<RELEASE_ARCH>.tar.gz | tar -x && chmod +x run && mv run ~/go/bin
 
-### **Using Go:**
+### Using Go
 
 If you already use go and have it installed, you can install RUN with the go command line tool.
 
     $ go install github.com/amonks/run/cmd/run@latest
 
+## Quick Start
+
+Follow these steps to get started with RUN:
+
+1. **Create a `tasks.toml` file**
+    
+        $ touch tasks.toml
+
+2. **Define a task in that `tasks.toml`**
```

Comment:
review note:
this replaces a couple different taskflie examples from the current readme

## first

```toml
# ./tasks.toml

[[task]]
  id = "build"
  type = "short"
  dependencies = ["css/build", "js/build"]
```

```toml
# ./css/tasks.toml

[[task]]
  id = "build"
  type = "short"
  watch = ["src.css"]
  cmd = """
    echo "Building CSS"
    build-css src.css > dist.css
    echo "done"
  """
```

## second

```toml
[[task]]
  id = "dev"
  type = "long"
  dependencies = ["simulate-coding"]
  triggers = ["build-css", "build-js"]
  watch = ["server-config.json"]
  env = {
    KEY="some value"
    OTHER_KEY="some other value"
  }
  cmd = """
    echo "dev-server running at http://localhost:3000"
    while true; do sleep 1; done
  """
```

Reply:
review:

the first existing example seems to emphasize modular taskfiles, which is a pretty niche feature and inappropriate to present as the first example

the second existing example seems to emphasize using every -other- feature besides modular taskfiles, which is also not great as an example

the example in this diff emphasizes using as few features as possible, which makes for a great quickstart! big improvement.

***

I wonder if a "short" or a "long" task would make for a better example — I feel like run shows the most value in "long" mode, but also your task configuration has to be a bit more complex to really illustrate that value, which might be worse for quickstart. Tradeoffs in all things.

Reply:
This makes sense to me. I think my goal here was to have some sort of "kitchen sink" task to reference in other places, but didn't end up needing to reference it really. and I agree that squashing the two example tasks into this one isn't adding value.

re short and long tasks, i almost feel like either a long or short task would work in the hello world exaample, and that it's really deps, triggers, and watches that add complexity into things.